### ISSUE TITLE
 Updating screen events  NullPointerException

### ISSUE THREAD

**Issue by ColorfulGhost** [micdoodle8/Galacticraft#3621]
Minecraft version: 1.12.2 
Galacticraft version: MicdoodleCore-1.12.2-4.0.1.187

```
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
	at micdoodle8.mods.galacticraft.core.recipe.NasaWorkbenchRecipe.checkItemEquals(NasaWorkbenchRecipe.java:40)
	at micdoodle8.mods.galacticraft.core.recipe.NasaWorkbenchRecipe.matches(NasaWorkbenchRecipe.java:29)
	at micdoodle8.mods.galacticraft.planets.mars.util.RecipeUtilMars.findMatchingSpaceshipT3Recipe(RecipeUtilMars.java:48)
	at micdoodle8.mods.galacticraft.planets.asteroids.inventory.ContainerSchematicTier3Rocket.func_75130_a(ContainerSchematicTier3Rocket.java:105)
	at micdoodle8.mods.galacticraft.planets.asteroids.inventory.InventorySchematicTier3Rocket.func_70299_a(InventorySchematicTier3Rocket.java:77)
	at net.minecraft.inventory.Slot.func_75215_d(Slot.java:70)
	at micdoodle8.mods.galacticraft.planets.asteroids.inventory.ContainerSchematicTier3Rocket.mergeOneItemTestValid(ContainerSchematicTier3Rocket.java:222)
	at micdoodle8.mods.galacticraft.planets.asteroids.inventory.ContainerSchematicTier3Rocket.func_82846_b(ContainerSchematicTier3Rocket.java:149)
	at net.minecraft.inventory.Container.func_184996_a(Container.java:246)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187098_a(PlayerControllerMP.java:566)
	at net.minecraft.client.gui.inventory.GuiContainer.func_184098_a(GuiContainer.java:638)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java:392)
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:533)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:501)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1759)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1098)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

```

**Comment by MJRLegends** [comment]
Do you have Galacticraft tweaker mod installed

**Comment by micdoodle8** [comment]
Definitely fixed with b39f44bae546653b95c85cde39a7f3dc0154921e

(Build 190+)
